src/testRunner/unittests/tsc/helpers.ts

```diff
@@ -71,15 +62,26 @@ namespace ts {
             }
         };
     }
+    export interface TscCompileLikeBase extends VerifyTscCompileLike {
+        buildKind?: BuildKind; // Should be defined for tsc --b
 
-    export function tscCompile(input: TscCompile) {
+        modifyFs?: (fs: vfs.FileSystem) => void;
+        disableUseFileVersionAsSignature?: boolean;
+        environmentVariables?: Record<string, string>;
+    }
+
+    export interface TscCompileLike extends TscCompileLikeBase {

```

Comment:
What does "tsc compile-like" mean?  What's like a tsc compile without being one?

Reply:
tscCompileLike is => Initialize FS, Call the worker callback to do custom action and then baseline.. 
tscCompile = tscCompileLike with worker executing commandline just like what command line tsc does.

Reply:
if you suggest names that make more sense i can update...

Reply:
I see, so it's not about an object that is "like" a tsc compile but indicates that the _test_ does something like run tsc?

Reply:
yes

Reply:
Personally, I would find it clearer if they all had a "Test" suffix, but it might be good to get a second opinion from someone more familiar with the code.

Reply:
And it wouldn't hurt to put your comments above in the code.